### PR TITLE
Remove templated variables in migrations

### LINKED ISSUE #3034: Migrations with default template values doen't work
### Steps to reproduce

1. Execute the auto-migration from Cobbler V 3.3.0
2. See that `bind_zonefile_path` has the wrong value in `settings.yaml`

### Expected behavior

The setting has the correct value.

### Cobbler version

````paste below
main
````

### Operating system

openSUSE Leap

### Cobbler log

````paste below
````

### Additional information

Discovered while working on #3004

### PR DESCRIPTION
**Pull request by SchoolGuy**
Fixes #3034

TODO:

- [x] Create test that shows that this works now.
- [x] Cleanup `V3.3.1.py`

### CODE REVIEW

**Comment by SchoolGuy** [comment]
So on my openSUSE Leap based container this is working but it seems openSUSE Tumbleweed has problems at the moment. I will investigate but this shouldn't prevent the review of this PR.

File: cobbler/settings/migrations/V3_3_1.py [hunk 1]
```diff
@@ -108,12 +108,12 @@
...
             ],
         ): list,
-        Optional("bootloaders_shim_folder", default="@@shim_folder@@"): str,
-        Optional("bootloaders_shim_file", default="@@shim_file@@"): str,
-        Optional("bootloaders_ipxe_folder", default="@@ipxe_folder@@"): str,
-        Optional("syslinux_dir", default="@@syslinux_dir@@"): str,
-        Optional("syslinux_memdisk_folder", default="@@memdisk_folder@@"): str,
-        Optional("syslinux_pxelinux_folder", default="@@pxelinux_folder@@"): str,
+        Optional("bootloaders_shim_folder", default="/usr/share/efi/*/"): str,
+        Optional("bootloaders_shim_file", default=r"shim\.efi"): str,
```
Comment (vzhestkov) [inline cobbler/settings/migrations/V3_3_1.py:112]: Is it really expected to have the regexp in this variable?

File: cobbler/settings/migrations/V3_3_1.py [hunk 2]
```diff
@@ -273,11 +273,11 @@ def migrate(settings: dict) -> dict:
         "ldap_tls_cacertdir": "",
         "ldap_tls_reqcert": "hard",
         "ldap_tls_cipher_suite": "",
-        "bootloaders_shim_folder": "@@shim_folder@@",
-        "bootloaders_shim_file": "@@shim_file@@",
-        "bootloaders_ipxe_folder": "@@ipxe_folder@@",
-        "syslinux_memdisk_folder": "@@memdisk_folder@@",
-        "syslinux_pxelinux_folder": "@@pxelinux_folder@@",
+        "bootloaders_shim_folder": "/usr/share/efi/*/",
+        "bootloaders_shim_file": r"shim\.efi",
```
Comment (vzhestkov) [inline cobbler/settings/migrations/V3_3_1.py:277]: The same question as before here.

File: cobbler/settings/migrations/V3_4_0.py [hunk 3]
```diff
@@ -106,12 +106,12 @@
...
             ],
         ): list,
-        Optional("bootloaders_shim_folder", default="@@shim_folder@@"): str,
-        Optional("bootloaders_shim_file", default="@@shim_file@@"): str,
-        Optional("bootloaders_ipxe_folder", default="@@ipxe_folder@@"): str,
-        Optional("syslinux_dir", default="@@syslinux_dir@@"): str,
-        Optional("syslinux_memdisk_folder", default="@@memdisk_folder@@"): str,
-        Optional("syslinux_pxelinux_folder", default="@@pxelinux_folder@@"): str,
+        Optional("bootloaders_shim_folder", default="/usr/share/efi/*/"): str,
+        Optional("bootloaders_shim_file", default=r"shim\.efi"): str,
```
Comment (vzhestkov) [inline cobbler/settings/migrations/V3_4_0.py:110]: And here

**Review by vzhestkov — COMMENTED** [review]
Not yet finished reviewing. Have to switch to other urgent task. Just some questions for now below...

File: cobbler/settings/migrations/V3_3_1.py (hunk 1 above)
Comment (vzhestkov) [reply cobbler/settings/migrations/V3_3_1.py:112]: Ah, ok, make sense

**Review by vzhestkov — APPROVED** [review]
Looks good. Hope the tests after the segfault with ldap test are not failing.

**Pull request merged**